src/cmds/fs/cat.c

```diff
@@ -43,26 +50,36 @@ int main(int argc, char **argv) {
 		}
 	}
 
-	if (NULL == (fd = fopen(argv[argc - 1], "r"))) {
-		printf("Can't open file %s\n", argv[argc - 1]);
-		return -errno;
-	}
+	for (no_of_arg = 1; no_of_arg < argc; no_of_arg++) {
+		if (strcmp ("-n", file_names[no_of_arg]) != 0) {
```

Comment:
Please follow our [code style](https://github.com/embox/embox/wiki/Code-Style), we don't place space between function name and opening parenthesis.